src/wallet/wallet.cpp

```diff
@@ -3809,7 +3809,7 @@ void CWallet::GetFilteredNotes(
 
 /* Find unspent notes filtered by payment address, min depth and max depth */
 void CWallet::GetUnspentFilteredNotes(
-    std::vector<CUnspentNotePlaintextEntry>& outEntries,
+    std::vector<CUnspentSproutNotePlaintextEntry>& outEntries,
```

Comment:
This will either be reverted once we add Sapling notes, or we'll add `outSaplingEntries`.